Platform-Linux/1-Ansible/ansible-jumpbox-connection.md

```diff
@@ -0,0 +1,40 @@
+There are multiple methods. The method I will be describing will be a method using the inventory file.
+All commands will be in the inventory file on the ansible controller
+
+# 1. Define a proxy/jumpbox host
```

Comment:
I would make this a h2 header `##` as I atleast relate the the h1 `#` to the document titles.